{
  "uid" : "bbe7c25a3a42d611",
  "name" : "TC 17.01.27 Verify all fields got cleared after the failed attempt to reset password",
  "fullName" : "tests.test_forgot_your_password.TestForgotYourPasswordFunctionality#test_17_01_27_fields_are_cleared_after_failed_attempt_to_reset_password",
  "historyId" : "329c5479a6abab9e833ef4806ddb95bd",
  "time" : {
    "start" : 1697333253497,
    "stop" : 1697333258626,
    "duration" : 5129
  },
  "description" : "Check if all the fields are getting cleared after failed attempt to reset password with invalid captcha",
  "descriptionHtml" : "<p>Check if all the fields are getting cleared after failed attempt to reset password with invalid captcha</p>\n",
  "status" : "broken",
  "statusMessage" : "selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x55f5890a66b3 <unknown>\n#1 0x55f588d7c1e7 <unknown>\n#2 0x55f588dc3566 <unknown>\n#3 0x55f588dc3651 <unknown>\n#4 0x55f588e00a74 <unknown>\n#5 0x55f588de50ed <unknown>\n#6 0x55f588dfe322 <unknown>\n#7 0x55f588de4e93 <unknown>\n#8 0x55f588db7934 <unknown>\n#9 0x55f588db871e <unknown>\n#10 0x55f58906bcb8 <unknown>\n#11 0x55f58906fbf0 <unknown>\n#12 0x55f58907a19c <unknown>\n#13 0x55f589070808 <unknown>\n#14 0x55f58903d27f <unknown>\n#15 0x55f589094e88 <unknown>\n#16 0x55f589095059 <unknown>\n#17 0x55f5890a5843 <unknown>\n#18 0x7ff771a94b43 <unknown>",
  "statusTrace" : "self = <test_forgot_your_password.TestForgotYourPasswordFunctionality object at 0x7fe0b137f790>\ndriver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"8ad0c1bc963b9fc5cbd556ab37728cd1\")>\nforgot_psw_page = <pages.forgot_your_password_page.ForgotYourPasswordPage object at 0x7fe0b09d8c50>\n\n    @allure.title('TC 17.01.27 Verify all fields got cleared after the failed attempt to reset password')\n    def test_17_01_27_fields_are_cleared_after_failed_attempt_to_reset_password(self, driver, forgot_psw_page):\n        \"\"\"Check if all the fields are getting cleared after failed attempt to reset password with invalid captcha\"\"\"\n>       forgot_psw_page.fill_in_email_field(credentials['valid_email'])\n\ntests/test_forgot_your_password.py:214: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npages/forgot_your_password_page.py:134: in fill_in_email_field\n    self.fill_in_field(self.check_forgot_psw_email_field_is_clickable(), email)\npages/forgot_your_password_page.py:36: in check_forgot_psw_email_field_is_clickable\n    return self.element_is_clickable(self.locators.EMAIL)\npages/base_page.py:80: in element_is_clickable\n    return wait(self.driver, timeout).until(EC.element_to_be_clickable(locator))\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"8ad0c1bc963b9fc5cbd556ab37728cd1\")>\nmethod = <function element_to_be_clickable.<locals>._predicate at 0x7fe0b09900e0>\nmessage = ''\n\n    def until(self, method, message: str = \"\"):\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       #0 0x55f5890a66b3 <unknown>\nE       #1 0x55f588d7c1e7 <unknown>\nE       #2 0x55f588dc3566 <unknown>\nE       #3 0x55f588dc3651 <unknown>\nE       #4 0x55f588e00a74 <unknown>\nE       #5 0x55f588de50ed <unknown>\nE       #6 0x55f588dfe322 <unknown>\nE       #7 0x55f588de4e93 <unknown>\nE       #8 0x55f588db7934 <unknown>\nE       #9 0x55f588db871e <unknown>\nE       #10 0x55f58906bcb8 <unknown>\nE       #11 0x55f58906fbf0 <unknown>\nE       #12 0x55f58907a19c <unknown>\nE       #13 0x55f589070808 <unknown>\nE       #14 0x55f58903d27f <unknown>\nE       #15 0x55f589094e88 <unknown>\nE       #16 0x55f589095059 <unknown>\nE       #17 0x55f5890a5843 <unknown>\nE       #18 0x7ff771a94b43 <unknown>\n\n/opt/hostedtoolcache/Python/3.11.5/x64/lib/python3.11/site-packages/selenium/webdriver/support/wait.py:95: TimeoutException",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ {
    "name" : "_session_faker",
    "time" : {
      "start" : 1697332188525,
      "stop" : 1697332188634,
      "duration" : 109
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "forgot_psw_page",
    "time" : {
      "start" : 1697333253238,
      "stop" : 1697333253496,
      "duration" : 258
    },
    "status" : "passed",
    "steps" : [ {
      "name" : "Open a browser",
      "time" : {
        "start" : 1697333253239,
        "stop" : 1697333253496,
        "duration" : 257
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false
    } ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 1,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : true
  }, {
    "name" : "driver",
    "time" : {
      "start" : 1697333252067,
      "stop" : 1697333253238,
      "duration" : 1171
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "clear_allure_result_folder",
    "time" : {
      "start" : 1697332188634,
      "stop" : 1697332188634,
      "duration" : 0
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  } ],
  "testStage" : {
    "description" : "Check if all the fields are getting cleared after failed attempt to reset password with invalid captcha",
    "status" : "broken",
    "statusMessage" : "selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x55f5890a66b3 <unknown>\n#1 0x55f588d7c1e7 <unknown>\n#2 0x55f588dc3566 <unknown>\n#3 0x55f588dc3651 <unknown>\n#4 0x55f588e00a74 <unknown>\n#5 0x55f588de50ed <unknown>\n#6 0x55f588dfe322 <unknown>\n#7 0x55f588de4e93 <unknown>\n#8 0x55f588db7934 <unknown>\n#9 0x55f588db871e <unknown>\n#10 0x55f58906bcb8 <unknown>\n#11 0x55f58906fbf0 <unknown>\n#12 0x55f58907a19c <unknown>\n#13 0x55f589070808 <unknown>\n#14 0x55f58903d27f <unknown>\n#15 0x55f589094e88 <unknown>\n#16 0x55f589095059 <unknown>\n#17 0x55f5890a5843 <unknown>\n#18 0x7ff771a94b43 <unknown>",
    "statusTrace" : "self = <test_forgot_your_password.TestForgotYourPasswordFunctionality object at 0x7fe0b137f790>\ndriver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"8ad0c1bc963b9fc5cbd556ab37728cd1\")>\nforgot_psw_page = <pages.forgot_your_password_page.ForgotYourPasswordPage object at 0x7fe0b09d8c50>\n\n    @allure.title('TC 17.01.27 Verify all fields got cleared after the failed attempt to reset password')\n    def test_17_01_27_fields_are_cleared_after_failed_attempt_to_reset_password(self, driver, forgot_psw_page):\n        \"\"\"Check if all the fields are getting cleared after failed attempt to reset password with invalid captcha\"\"\"\n>       forgot_psw_page.fill_in_email_field(credentials['valid_email'])\n\ntests/test_forgot_your_password.py:214: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npages/forgot_your_password_page.py:134: in fill_in_email_field\n    self.fill_in_field(self.check_forgot_psw_email_field_is_clickable(), email)\npages/forgot_your_password_page.py:36: in check_forgot_psw_email_field_is_clickable\n    return self.element_is_clickable(self.locators.EMAIL)\npages/base_page.py:80: in element_is_clickable\n    return wait(self.driver, timeout).until(EC.element_to_be_clickable(locator))\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"8ad0c1bc963b9fc5cbd556ab37728cd1\")>\nmethod = <function element_to_be_clickable.<locals>._predicate at 0x7fe0b09900e0>\nmessage = ''\n\n    def until(self, method, message: str = \"\"):\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       #0 0x55f5890a66b3 <unknown>\nE       #1 0x55f588d7c1e7 <unknown>\nE       #2 0x55f588dc3566 <unknown>\nE       #3 0x55f588dc3651 <unknown>\nE       #4 0x55f588e00a74 <unknown>\nE       #5 0x55f588de50ed <unknown>\nE       #6 0x55f588dfe322 <unknown>\nE       #7 0x55f588de4e93 <unknown>\nE       #8 0x55f588db7934 <unknown>\nE       #9 0x55f588db871e <unknown>\nE       #10 0x55f58906bcb8 <unknown>\nE       #11 0x55f58906fbf0 <unknown>\nE       #12 0x55f58907a19c <unknown>\nE       #13 0x55f589070808 <unknown>\nE       #14 0x55f58903d27f <unknown>\nE       #15 0x55f589094e88 <unknown>\nE       #16 0x55f589095059 <unknown>\nE       #17 0x55f5890a5843 <unknown>\nE       #18 0x7ff771a94b43 <unknown>\n\n/opt/hostedtoolcache/Python/3.11.5/x64/lib/python3.11/site-packages/selenium/webdriver/support/wait.py:95: TimeoutException",
    "steps" : [ {
      "name" : "Fill in Email field",
      "time" : {
        "start" : 1697333253497,
        "stop" : 1697333258626,
        "duration" : 5129
      },
      "status" : "broken",
      "statusMessage" : "selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x55f5890a66b3 <unknown>\n#1 0x55f588d7c1e7 <unknown>\n#2 0x55f588dc3566 <unknown>\n#3 0x55f588dc3651 <unknown>\n#4 0x55f588e00a74 <unknown>\n#5 0x55f588de50ed <unknown>\n#6 0x55f588dfe322 <unknown>\n#7 0x55f588de4e93 <unknown>\n#8 0x55f588db7934 <unknown>\n#9 0x55f588db871e <unknown>\n#10 0x55f58906bcb8 <unknown>\n#11 0x55f58906fbf0 <unknown>\n#12 0x55f58907a19c <unknown>\n#13 0x55f589070808 <unknown>\n#14 0x55f58903d27f <unknown>\n#15 0x55f589094e88 <unknown>\n#16 0x55f589095059 <unknown>\n#17 0x55f5890a5843 <unknown>\n#18 0x7ff771a94b43 <unknown>\n\n",
      "statusTrace" : "  File \"/opt/hostedtoolcache/Python/3.11.5/x64/lib/python3.11/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/luma_project/luma_project/pages/forgot_your_password_page.py\", line 134, in fill_in_email_field\n    self.fill_in_field(self.check_forgot_psw_email_field_is_clickable(), email)\n                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.5/x64/lib/python3.11/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/luma_project/luma_project/pages/forgot_your_password_page.py\", line 36, in check_forgot_psw_email_field_is_clickable\n    return self.element_is_clickable(self.locators.EMAIL)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.5/x64/lib/python3.11/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/luma_project/luma_project/pages/base_page.py\", line 80, in element_is_clickable\n    return wait(self.driver, timeout).until(EC.element_to_be_clickable(locator))\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.5/x64/lib/python3.11/site-packages/selenium/webdriver/support/wait.py\", line 95, in until\n    raise TimeoutException(message, screen, stacktrace)\n",
      "steps" : [ {
        "name" : "Check FYP Email field is clickable",
        "time" : {
          "start" : 1697333253497,
          "stop" : 1697333258625,
          "duration" : 5128
        },
        "status" : "broken",
        "statusMessage" : "selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x55f5890a66b3 <unknown>\n#1 0x55f588d7c1e7 <unknown>\n#2 0x55f588dc3566 <unknown>\n#3 0x55f588dc3651 <unknown>\n#4 0x55f588e00a74 <unknown>\n#5 0x55f588de50ed <unknown>\n#6 0x55f588dfe322 <unknown>\n#7 0x55f588de4e93 <unknown>\n#8 0x55f588db7934 <unknown>\n#9 0x55f588db871e <unknown>\n#10 0x55f58906bcb8 <unknown>\n#11 0x55f58906fbf0 <unknown>\n#12 0x55f58907a19c <unknown>\n#13 0x55f589070808 <unknown>\n#14 0x55f58903d27f <unknown>\n#15 0x55f589094e88 <unknown>\n#16 0x55f589095059 <unknown>\n#17 0x55f5890a5843 <unknown>\n#18 0x7ff771a94b43 <unknown>\n\n",
        "statusTrace" : "  File \"/opt/hostedtoolcache/Python/3.11.5/x64/lib/python3.11/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/luma_project/luma_project/pages/forgot_your_password_page.py\", line 36, in check_forgot_psw_email_field_is_clickable\n    return self.element_is_clickable(self.locators.EMAIL)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.5/x64/lib/python3.11/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/luma_project/luma_project/pages/base_page.py\", line 80, in element_is_clickable\n    return wait(self.driver, timeout).until(EC.element_to_be_clickable(locator))\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.5/x64/lib/python3.11/site-packages/selenium/webdriver/support/wait.py\", line 95, in until\n    raise TimeoutException(message, screen, stacktrace)\n",
        "steps" : [ {
          "name" : "Find clickable elements",
          "time" : {
            "start" : 1697333253497,
            "stop" : 1697333258625,
            "duration" : 5128
          },
          "status" : "broken",
          "statusMessage" : "selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x55f5890a66b3 <unknown>\n#1 0x55f588d7c1e7 <unknown>\n#2 0x55f588dc3566 <unknown>\n#3 0x55f588dc3651 <unknown>\n#4 0x55f588e00a74 <unknown>\n#5 0x55f588de50ed <unknown>\n#6 0x55f588dfe322 <unknown>\n#7 0x55f588de4e93 <unknown>\n#8 0x55f588db7934 <unknown>\n#9 0x55f588db871e <unknown>\n#10 0x55f58906bcb8 <unknown>\n#11 0x55f58906fbf0 <unknown>\n#12 0x55f58907a19c <unknown>\n#13 0x55f589070808 <unknown>\n#14 0x55f58903d27f <unknown>\n#15 0x55f589094e88 <unknown>\n#16 0x55f589095059 <unknown>\n#17 0x55f5890a5843 <unknown>\n#18 0x7ff771a94b43 <unknown>\n\n",
          "statusTrace" : "  File \"/opt/hostedtoolcache/Python/3.11.5/x64/lib/python3.11/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/luma_project/luma_project/pages/base_page.py\", line 80, in element_is_clickable\n    return wait(self.driver, timeout).until(EC.element_to_be_clickable(locator))\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.5/x64/lib/python3.11/site-packages/selenium/webdriver/support/wait.py\", line 95, in until\n    raise TimeoutException(message, screen, stacktrace)\n",
          "steps" : [ ],
          "attachments" : [ ],
          "parameters" : [ {
            "name" : "locator",
            "value" : "('css selector', '#email_address')"
          }, {
            "name" : "timeout",
            "value" : "5"
          } ],
          "stepsCount" : 0,
          "attachmentsCount" : 0,
          "shouldDisplayMessage" : true,
          "hasContent" : true
        } ],
        "attachments" : [ ],
        "parameters" : [ ],
        "stepsCount" : 1,
        "attachmentsCount" : 0,
        "shouldDisplayMessage" : false,
        "hasContent" : true
      } ],
      "attachments" : [ ],
      "parameters" : [ {
        "name" : "email",
        "value" : "'test130@mailinator.com'"
      } ],
      "stepsCount" : 2,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : true
    } ],
    "attachments" : [ {
      "uid" : "2c74adfe9034c498",
      "name" : "Screenshot",
      "source" : "2c74adfe9034c498.png",
      "type" : "image/png",
      "size" : 14924
    }, {
      "uid" : "213d13cec569dda7",
      "name" : "HTML source",
      "source" : "213d13cec569dda7.html",
      "type" : "text/html",
      "size" : 571
    } ],
    "parameters" : [ ],
    "stepsCount" : 3,
    "attachmentsCount" : 2,
    "shouldDisplayMessage" : true,
    "hasContent" : true
  },
  "afterStages" : [ {
    "name" : "driver::0",
    "time" : {
      "start" : 1697333258810,
      "stop" : 1697333258871,
      "duration" : 61
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "clear_allure_result_folder::0",
    "time" : {
      "start" : 1697335158583,
      "stop" : 1697335158584,
      "duration" : 1
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  } ],
  "labels" : [ {
    "name" : "epic",
    "value" : "Forgot Your Password Functionality"
  }, {
    "name" : "parentSuite",
    "value" : "tests"
  }, {
    "name" : "suite",
    "value" : "test_forgot_your_password"
  }, {
    "name" : "subSuite",
    "value" : "TestForgotYourPasswordFunctionality"
  }, {
    "name" : "host",
    "value" : "fv-az163-423"
  }, {
    "name" : "thread",
    "value" : "3041-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "tests.test_forgot_your_password"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Test defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "tags" : [ ]
  },
  "source" : "bbe7c25a3a42d611.json",
  "parameterValues" : [ ]
}